### PR TITLE
UI bug fixes and improvements related to console settings

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

### Related Issues

- N/A

### Related PRs

- N/A

### Checklist

- [ ] e2e cypress tests locally verified. (for internal contributers)
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Relevant backend changes deployed and verified
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any)
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

File: features/admin.core.v1/components/header.tsx [hunk 1]
```diff
@@ -150,7 +177,11 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (props: HeaderPro
         (organizationType === OrganizationType.SUPER_ORGANIZATION ||
             organizationType === OrganizationType.FIRST_LEVEL_ORGANIZATION ||
             organizationType === OrganizationType.SUBORGANIZATION ||
-            organizationConfigs.showSwitcherInTenants) && hasOrganizationReadPermission
+            organizationConfigs.showSwitcherInTenants) &&
+        hasOrganizationReadPermission
```
Comment (pavinduLakshan) [inline features/admin.core.v1/components/header.tsx:181]: Need to enable the org switcher for everyone in the primary userstore with console access

Comment (pavinduLakshan) [reply features/admin.core.v1/components/header.tsx]: Fixed with [3b2069d](https://github.com/wso2/identity-apps/pull/7141/commits/3b2069d00f35b0b67efa66c0c54556103aaac65f)

File: features/admin.extensions.v1/configs/models/documentation.ts [hunk 2]
```diff
@@ -616,6 +623,7 @@ export interface DocumentationLinksExtensionInterface {
          * Documentation links for account recovery elements.
          */
         accountRecovery: AccountRecoveryDocumentationLinksInterface;
+        administrators: AdministratorsDocumentationLinksInterface;
```
Comment (JayaShakthi97) [inline features/admin.extensions.v1/configs/models/documentation.ts:626]: Let's add a doc comment here, as other fields has that.

Comment (pavinduLakshan) [reply features/admin.extensions.v1/configs/models/documentation.ts:629]: Fixed with [4735aca](https://github.com/wso2/identity-apps/pull/7141/commits/4735aca33a59a39f2d466a38711df8632b37998e)

File: features/admin.roles.v2/utils/role-management-utils.ts [hunk 3]
```diff
@@ -130,10 +131,14 @@ export class RoleManagementUtils {
      * @param nameWithUserstore - name with userstore
      * @returns - userstore
      */
-    public static getUserStore = (nameWithUserstore: string): string =>
-        nameWithUserstore?.split("/").length > 1
+    public static getUserStore = (nameWithUserstore: string): string => {
+        const isDefaultPrimaryUserStoreEnabled: boolean =
+            isFeatureEnabled(store.getState().config?.ui?.features?.userStores,"PRIMARY");
```
[the remark this answers is not in the document]
Comment (pavinduLakshan) [reply features/admin.roles.v2/utils/role-management-utils.ts]: Fixed with [5d28a13](https://github.com/wso2/identity-apps/pull/7141/commits/5d28a1325a6d67c391bee732bba61d8855e1d8d0)

**Review by NipuniBhagya — APPROVED** [review]

**Pull request merged**